src/transformers/generation_utils.py

```diff
@@ -407,18 +694,37 @@ def generate(
                 conditioned on the previously generated tokens :obj:`inputs_ids` and the batch ID :obj:`batch_id`. This
                 argument is useful for constrained generation conditioned on the prefix, as described in
                 `Autoregressive Entity Retrieval <https://arxiv.org/abs/2010.00904>`__.
+            output_attentions (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return attention weights in output
+            output_hidden_states (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return hidden states in output
+            output_scores (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return the scores in output
+            return_dict_in_generate (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return a dict if the generation method supports it
+
             model_kwargs:
                 Additional model specific kwargs will be forwarded to the :obj:`forward` function of the model. If the
                 model is an Encoder-Decoder model, encoder specific kwargs should not be prefixed and decoder specific
                 kwargs should be prefixed with `decoder_`.
 
+
         Return:
-            :obj:`torch.LongTensor` of shape :obj:`(batch_size * num_return_sequences, sequence_length)`: The generated
-            sequences. The second dimension (sequence_length) is either equal to :obj:`max_length` or shorter if all
-            batches finished early due to the :obj:`eos_token_id`.
+            If model is `not` an Encoder-Decoder model (``model.config.is_encoder_decoder=False``), either one of
```

Comment:
```suggestion
            If model is `not` an encoder-decoder model (``model.config.is_encoder_decoder=False``), either one of
```
(encoder-decoder was spelled without capital before, I don't really care which one is picked but I like consistency ;-) )